README.md

```diff
@@ -1,4 +1,14 @@
-POC of new oVirt User Portal.
+# User Portal for oVirt
+Fancy looking UI for standard (non-admin) oVirt users.
```

Comment:
Suggest "NG" since we use that a lot in oVirt.

-1 to "Fancy looking" for sure.